.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
```

Comment:
Do we just want to do it on master or every branch in the repository?

Reply:
From what I've understood, if you don't specify the branch, it will run the CI twice when you commit on the PRs.
See https://github.com/dandavison/delta/pull/467

Reply:
ah ok, well then lgtm.